docs/pages/versions/unversioned/guides/adhoc-builds.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Installing a Custom iOS Expo Client
+---
+
+![Build Overview](/static/images/adhoc-builds-overview.gif)
+
+Build and install a custom version of the Expo Client (link) with your own Apple Credentials using our build service. This custom version of the Expo Client contains features that were previously only available on the Android versions. Our build service will prepare your custom Expo Client, and you can install it to your iOS device directly from our website.
+
+## 0. Prerequisites
+
+You'll need an [Apple Developer Account](https://developer.apple.com/programs)
+
+## 1. Install Expo CLI
+
+Expo CLI is the tool for developing and building Expo apps. Run `npm install -g expo-cli` (or `yarn global add expo-cli`) to get it.
+
+If you haven't created an Expo account before, you'll be asked to create one when running the build command.
+
+**Windows users** must have WSL enabled. We recommend picking Ubuntu from the Windows Store. Be sure
```

Comment:
may be best to link to a WSL installation/setup guide rather than including the info about how to enable it in-line here